pyproject.toml

```diff
@@ -269,7 +269,7 @@ force-exclude = '''
 [tool.codespell]
 skip = "*.dat,*.fits,*.hdr,*.xml,*egg*,*extern,.git,.tox,_build,fitshdr.htest_groups.py,venv,_*.c,wcs.h,lin.h,tab.h,spc.h,cython*"
 # The following list of words for codespell to ignore may contain some
-# mispellings that should be revisited and fixed in the future.
+# misspellings that should be revisited and fixed in the future.
```

Comment:
Oops!  It appears I added a misspelling in my previous PR to fix misspellings. 👀 😅